lib/std/Build/RunStep.zig

```diff
@@ -72,6 +72,10 @@ captured_stderr: ?*Output = null,
 
 has_side_effects: bool = false,
 
+/// Set this to true if RunStep runs a zig test.
+/// See https://github.com/ziglang/zig/issues/15104.
```

Comment:
I don't think this should link to an issue that the PR closes.

Reply:
> I don't think this should link to an issue that the PR closes.

I can improve the documentation, but I'm not sure how to explain why the new that is necessary.

An alternative solution is to simply **never** close `stdin`, and instead ignore it.  However I'm not not sure if there is a valid reason for the current implementation.

Thanks.